### DISCUSSION TITLE
View CSV  Error Response | Bulk Data Imports

### DISCUSSION THREAD

**Discussion by HunterJacobs** [burkybang/SuiteAdvanced-Feedback#186] [Ideas]
Would it be possible to have it where you can view the CSV response in Netsuite?

Usually I will run into a simple issue and this would save time trying to resolve instead of having to download the file each time it has an error. Even if it just had a few rows most of my issues or something simple and repetitive. Or even if it gave a summary of the issues in the file. Not sure if this is even possible though.

![image](https://github.com/user-attachments/assets/91fdb46f-1ca5-4b5f-a78e-7692c788f236)

**Comment by burkybang** [comment]
That's a great idea. I'll check it out.

**Reply by burkybang** [reply]
@HunterJacobs I've added this as a feature in the latest version!

**Reply by HunterJacobs** [reply]
You're the best! I saw the GitHub mention on the update. It works better than I'd have imagined it to and is really helpful.
